protocol/testing/e2e/gov/prices_test.go

```diff
@@ -140,6 +142,34 @@ func TestUpdateMarketParam(t *testing.T) {
 						genesisState.Params.VotingPeriod = &testapp.TestVotingPeriod
 					},
 				)
+				// Initialize marketmap module with genesis market.
+				testapp.UpdateGenesisDocWithAppStateForModule(
+					&genesis,
+					func(genesisState *marketmaptypes.GenesisState) {
+						markets := make(map[string]marketmaptypes.Market)
+						cp, _ := slinky.MarketPairToCurrencyPair(GENESIS_MARKET_PARAM.Pair)
+						ticker := marketmaptypes.Ticker{
```

Comment:
nit: Add a constant to `testutil/constants` for this market map market?

Reply:
This is derived from the GENESIS_MARKET_PARAM var defined in this file, so I think it makes sense to leave it like this